Earthfile

```diff
@@ -130,6 +131,14 @@ earth:
     SAVE ARTIFACT ./build/ldflags
     SAVE ARTIFACT build/earth AS LOCAL "build/$GOOS/$GOARCH/earth"
 
+earth-arm5:
+    COPY \
+        --build-arg GOARCH=arm \
+        --build-arg GOARM=5 \
+        --build-arg GO_EXTRA_LDFLAGS= \
+        +earth/* ./
```

Comment:
Within the earth target, should the `AS LOCAL` also use `build/linux/arm5/earth` rather than `build/linux/arm/earth`?

Reply:
that's a good point, I'll give that a try.